apps/golden-sample-app/src/index.html

```diff
@@ -8,6 +8,33 @@
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
+    <script type="text/javascript">
+      function trackingEventsFake (eventType, event) {
+        console.log('tracking event type:', eventType, event);
+      }
+
+      // Listening for click events
+      document.addEventListener('click', function (event) {
```

Comment:
@tcorral i like the idea of capturing clicks at the document level. My question though is there are situations when click is not triggered for example while selecting a dropdown option with a keyboard. 

- Do we want to log actions on different set of controls that does not trigger a click (select, input type="texts" and date)

Reply:
What we have been requested and what is more common it's to take care of navigation and click in links and buttons butI do think that what you are asking for will be something to resolve using custom events and maybe at the ui-ang level.